src/lib/viewers/doc/PresentationViewer.js

```diff
@@ -125,15 +129,6 @@ class PresentationViewer extends DocBaseViewer {
         return hasXOverflow || hasYOverflow;
     }
 
-    /**
-     * @override
-     */
-    handleScrollToAnnotation(data) {
```

Comment:
This is a breaking change and require another joint release with BUIE. We'll need to start considering the effect of cross-library changes like this as we move into release.